pywinauto/controls/uiawrapper.py

```diff
@@ -775,6 +775,24 @@ def menu_select(self, path, exact=False, ):
         menu = cc[0]
         menu.item_by_path(path, exact).select()
 
+    # -----------------------------------------------------------
+    def toolbar_select(self, path, exact=False):
+        """
+        Select a Toolbar item specified in the path.
+
+        The full path syntax is specified in:
+        :py:meth:`pywinauto.controls.uia_controls.ToolbarWrapper.item_by_path`
+        """
+        self.verify_actionable()
+
+        cc = self.children(control_type="ToolBar")
+        if not cc:
+            cc = self.descendants(control_type="ToolBar")
+            if not cc:
+                raise AttributeError("Can not find ToolBar item in {}".format(self.descendants()))
```

Comment:
Calling any wrapper method inside `raise SomeException` statement is risky! Because it may raise another exception and add more confusion. Please use general phrase like `"Cannot find any item with control_type=\"ToolBar\" in children and descendants"`.